### ISSUE TITLE
Change in particle data (Gizmo) from yt-3.2 to yt-3.3?

### ISSUE THREAD

**Issue by yt-fido** [yt-project/yt#1323]
Originally reported by: **Kyle Stewart (Bitbucket: [kstewart](https://bitbucket.org/kstewart), GitHub: [kstewart](https://github.com/kstewart))**

----------------------------------------

Apologies if this information is available in the reading list and I simply couldn't find it.

I recently went back to re-do some calculations I made last year using yt-3.2, and as I redo the exact same calculations, I keep finding some very different answers on calculations made purely from the particle fields in a Gizmo dataset.

Namely, if you look at ('PartType0','temperature') in the dataset using yt-3.2, and again using yt-3.3, (exact same calculation / same dataset / same region) you get a noticeably different distribution of particle temperatures.  I've reproduced the issue here with the test dataset: FIRE_M12i_ref11/snapshot_600.hdf5 by plotting the mass-weighted temperature distribution within 300 kpc of the galaxy (i.e. maximum value of "all_densitiy").

Did something drastic change in yt-3.3 about the way the *particle* data was read in?  Hopefully this isn't a dumb question but shouldn't the raw particle data for a particle-based dataset be identical?  Was there a bug in yt-3.2 that got fixed, perhaps?  Or has a bug been introduced?  

In short, since I'm re-doing some calculations and getting different answers, I'm just hoping to know which of them is correct...

Thanks so much!

Kyle

----------------------------------------
- Bitbucket: https://bitbucket.org/yt_analysis/yt/issue/1323
